api/src/pcapi/core/mails/transactional/bookings/booking_postponed_by_pro_to_beneficiary.py

```diff
@@ -0,0 +1,59 @@
+from typing import Union
+
+from babel.dates import format_date
+
+from pcapi.core import mails
+from pcapi.core.bookings.models import Booking
+from pcapi.core.mails.transactional.sendinblue_template_ids import SendinblueTransactionalEmailData
+from pcapi.core.mails.transactional.sendinblue_template_ids import TransactionalEmail
+from pcapi.models.feature import FeatureToggle
+from pcapi.utils.mailing import format_booking_hours_for_email
+from pcapi.utils.mailing import get_event_datetime
+
+
+def send_batch_booking_postponement_email_to_users(bookings: list[Booking]) -> list[bool]:
+    for booking in bookings:
+        send_booking_postponement_email_to_users(booking)
+
+
+def send_booking_postponement_email_to_users(booking: Booking) -> bool:
+    data = get_booking_postponed_by_pro_to_beneficiary_email_data(booking)
+    return mails.send(recipients=[booking.email], data=data)
+
+
+def get_booking_postponed_by_pro_to_beneficiary_email_data(
+    booking: Booking,
+) -> Union[dict, SendinblueTransactionalEmailData]:
+    stock = booking.stock
+    offer = stock.offer
+
+    if offer.isEvent:
+        event_date = format_date(get_event_datetime(stock), format="full", locale="fr")
+        event_hour = format_booking_hours_for_email(booking)
+    else:
+        event_date = None
+        event_hour = None
```

Comment:
Puisqu'on envoie `""` à Mailjet et SendinBlue dans le cas où l'offre n'est pas un événement, autant tout de suite définir la bonne valeur par défaut ici, ça évitera des if plus bas.

```suggestion
        event_date = ""
        event_hour = ""
```